app/views/shared/_user_info.html.erb

```diff
@@ -12,6 +12,10 @@
     <span class="text-hamers-red-400"><%= current_user.stickers.count %></span> stickers geplakt,
     <span class="text-hamers-red-400"><%= current_user.blogitems.count %></span> blogs geschreven en
     bent bij <span class="text-hamers-red-400"><%= Signup.where(event_id: Event.where(attendance: true).pluck(:id), user: current_user).count %></span>
-    borrels geweest.
+    borrels geweest, met een inschrijfpercentage van <span class="text-hamers-red-400"><%= current_user.drink_ratio %>%</span> in het afgelopen jaar.
```

Comment:
Afronden naar 1 decimaal?

Reply:
https://github.com/DispuutHamers/webapp/pull/460

Reply:
Voor 0 gekozen want mooier en precisie is toch niet nodig